spec/integration/find_options_spec.rb

```diff
@@ -187,4 +202,25 @@
       end
     end
   end
+
+  describe 'cursor type' do
+    let(:collection_options) do
+      { capped: true, size: 1000 }
+    end
+
+    context 'when cursor type is :tailable' do
+      it 'sets the cursor type to tailable' do
+        collection.find({}, cursor_type: :tailable).first
+        expect(find_command.command['tailable']).to be true
+      end
```

Comment:
Maybe worth checking that `awaitData` is nil/false here? Otherwise, the test would still pass even if `:tailable_await` were given, instead of `:tailable`.

Reply:
Good point, thank you! Added.